infrastructure/code/fake_code_client_scanner.go

```diff
@@ -292,7 +292,7 @@ func getSarifResponseJson2(filePath string) string {
 								"properties": {
 									"category": "wont-fix",
 									"expiration": "13 days",
-									"ignoredOn": "2024-02-23T16:08:25Z",
+									"ignoredOn": "Wed Jun 05 2024",
 									"ignoredBy": {
 										"name": "Neil M",
 										"email": "test@test.io"
```

Comment:
IDE-258

Reply:
Our REST APIs should be consistent on date time formatting, expressed in the OpenAPI spec as:

```
type: string
format: date-time
```

So the original was correct. This looks like a regression or bug in the API. Please open a bug with the API owner 🙏

Reply:
Oh... this is SARIF. Hmm. Consulting the SARIF spec.

Reply:
Looks like this is free-form metadata in the suppression object. I think we should still push for RFC3339 (unambigious, machine readable, consistent with our other APIs...)